resources/appsignal.rb.erb

```diff
@@ -0,0 +1,32 @@
+# AppSignal Ruby gem configuration
+# Visit our documentation for a list of all available configuration options.
+# https://docs.appsignal.com/ruby/configuration/options.html
+Appsignal.configure do |config|
+  config.activate_if_environment(<%= environments.map(&:inspect).join(", ") %>)
+  config.name = <%= app_name.inspect %>
+  # The application's Push API key
+  # We recommend removing this line and setting this option with the
+  # APPSIGNAL_PUSH_API_KEY environment variable instead.
+  # https://docs.appsignal.com/ruby/configuration/options.html#option-push_api_key
+  config.push_api_key = "<%= push_api_key %>"
+
+  # Actions that should not be monitored by AppSignal
+  # config.ignore_actions << "ApplicationController#isup"
+
+  # Errors that should not be recorded by AppSignal
+  # For more information see our docs:
+  # https://docs.appsignal.com/ruby/configuration/ignore-errors.html
+  # config.ignore_errors += [
+  #   "Exception",
+  #   "NoMemoryError",
+  #   "ScriptError",
+  #   "LoadError",
+  #   "NotImplementedError",
+  #   "SyntaxError",
+  #   "SecurityError",
+  #   "SignalException",
+  #   "Interrupt",
+  #   "SystemExit",
+  #   "SystemStackError"
+  # ]
```

Comment:
Do we want to suggest ignoring all of these out of the box? I wonder if people will uncomment this without thinking and accidentally miss on errors. Perhaps suggest something like:

```suggestion
  # config.ignore_errors += [
  #   "MyCustomError",
  # ]
```

Reply:
Sure. I just copied the template from the appsignal.yml file 🤷‍♂️